brainstormforce/ultimate-addons-for-gutenberg
# fix: Clicking on answer collapsed the faq.

### Description

Allow links to be clickable within FAQ sections.

### Types of changes




Bug fix (non-breaking change which fixes an issue).

### How has this been tested?

- Click on the Ans tab should be FAQ open.
- Click on the Question tab FAQ should be close.
- Click on the link it should be open.

### Checklist:
